### PR TITLE
Add Dockerfile config templates from laravel-docker, generate them as files, and create a testcase to m…

### PR DESCRIPTION
**Pull request by KTanAug21**
…ake sure they get generated

TLDR: we're porting the [fly-apps/laravel-docker](https://github.com/fly-apps/laravel-docker) functionality over to this package, so that this package may create a Dockerfile that does not use the custom image built by that repository, but instead use an upstream image instead( Which the Dockerfile the [fly-apps/laravel-docker](https://github.com/fly-apps/laravel-docker) repo creates, already does! )  

**WHAT AND WHY:**
So, in preparation of using an upstream base image in the Dockerfile generated by this package, this PR aims to prepare the necessary files needed by the Dockerfile generated by fly-apps/laravel-docker:
1. Copy over config files required by the Dockerfile generated by [fly-apps/laravel-docker](https://github.com/fly-apps/laravel-docker). 
2. Have the `generate` command render the config files needed by the new Dockerfile into the users project, temporarily through the use of the --upstream-base-image flag

For more reference on this update, see [discourse post](https://flyio.discourse.team/t/how-to-approach-securtiy-patches-for-fly-apps-laravel-docker/6484/7), see  [Github Issue  ](https://github.com/fly-apps/dockerfile-laravel/issues/42)

### CODE REVIEW

**Comment by KTanAug21** [comment]
Closing this, as a new PR has been created in replacement of this,[ see PR!](https://github.com/fly-apps/dockerfile-laravel/pull/44)
